nevergrad/optimization/experimentalvariants.py

```diff
@@ -273,11 +272,23 @@
 ).set_name("HCHAvgCauchyLHSSearch", register=True)
 
 # Split on top of competence map.
-MetaNGOpt10 = ConfSplitOptimizer(
+MetaNGOpt10 = opts.ConfSplitOptimizer(
     multivariate_optimizer=NGOpt10, monovariate_optimizer=NGOpt10, non_deterministic_descriptor=False
 ).set_name("MetaNGOpt10", register=True)
 
 # Multiple single runs for multi-objective optimization.
-NGOptSingle9 = MultipleSingleRuns(num_single_runs=9).set_name("NGOptSingle9", register=True)
-NGOptSingle16 = MultipleSingleRuns(num_single_runs=16).set_name("NGOptSingle16", register=True)
-NGOptSingle25 = MultipleSingleRuns(num_single_runs=25).set_name("NGOptSingle25", register=True)
+NGOptSingle9 = opts.MultipleSingleRuns(num_single_runs=9).set_name("NGOptSingle9", register=True)
+NGOptSingle16 = opts.MultipleSingleRuns(num_single_runs=16).set_name("NGOptSingle16", register=True)
+NGOptSingle25 = opts.MultipleSingleRuns(num_single_runs=25).set_name("NGOptSingle25", register=True)
+
+# noisy splitters.
+Noisy13Splits = opts.NoisySplitter(num_optims=13, discrete=False).set_name("Noisy13Splits", register=True)
+NoisyInfSplits = opts.NoisySplitter(num_optims=float("inf"), discrete=False).set_name(
+    "NoisyInfSplits", register=True
+)
+DiscreteNoisy13Splits = opts.NoisySplitter(num_optims=13, discrete=True).set_name(
+    "DiscreteNoisy13Splits", register=True
+)
+DiscreteNoisyInfSplits = opts.NoisySplitter(num_optims=float("inf"), discrete=True).set_name(
+    "DiscreteNoisyInfSplits", register=True
+)
```

Comment:
@teytaud both "Inf" optimizers are equivalent if I get it right, since they will use the monovariate optimizer instead.

Reply:
Correct.

Reply:
It's basically doing bandit + progressive widening then.